reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md

```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
 
 The Windows Compatibility Pack enables PowerShell Core to use **more than 1900 cmdlets that ship with Windows 10 October 2018 Update and Windows Server 2019**.
 
+## Support for Application Whitelisting
+
+PowerShell Core 6.1 has parity with Windows PowerShell 5.1 supporting [AppLocker](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/applocker/applocker-overview)
+and [Device Guard](https://docs.microsoft.com/en-us/windows/security/threat-protection/device-guard/introduction-to-device-guard-virtualization-based-security-and-windows-defender-application-control) application whitelisting.
+Application whitelisting allows granular control of what binaries are allowed to be executed used with PowerShell [Constrained Language mode](https://blogs.msdn.microsoft.com/powershell/2017/11/02/powershell-constrained-language-mode/).
```

Comment:
Perhaps:
Application white listing provides granular control of which binaries are allowed to run on a system.  PowerShell runs in Constrained Language mode when such a policy is enforced on a system, and only allows policy trusted script files to run in Full Language mode.

Reply:
Looks like this was merged. Please submit new PR, your feedback makes sense.